aten/src/ATen/native/native_functions.yaml

```diff
@@ -1859,6 +1865,7 @@
   dispatch:
     CUDA: cudnn_batch_norm
   autogen: cudnn_batch_norm.out
+  tags: maybe_aliasing_or_mutating
```

Comment:
@bdhirsh  This is non-CIA op with wrong schema. After @andrewor14 's work on consolidation, can we expect to never see this in PT2? In that case, i think it is ok to just not tag them and only tag CIA ones